### PR TITLE
Be more careful about getting enough synth.

### PR DESCRIPTION
**Pull request by phulin**

### CODE REVIEW

File: src/embezzler.ts [hunk 1]
```diff
@@ -316,13 +316,18 @@ export function embezzlerCount(): number {
...
+  const thumbRingMultiplier = have($item`mafia thumb ring`) ? 1.04 : 1;
   let turns;
   if (globalOptions.stopTurncount) turns = globalOptions.stopTurncount - myTurncount();
   else if (globalOptions.noBarf) turns = embezzlerCount();
-  else
-    turns =
-      (myAdventures() + (globalOptions.ascending && myInebriety() <= inebrietyLimit() ? 60 : 0)) *
-      (have($item`mafia thumb ring`) ? 1.04 : 1);
+  else {
+    turns = (myAdventures() + pantsgivingAdventures + nightcapAdventures) * thumbRingMultiplier;
```
Comment (horrible-little-slime) [inline src/embezzler.ts:329]: if we're rehauling this, we should account for magical sausages

File: src/embezzler.ts [hunk 2]
```diff
@@ -316,13 +316,18 @@ export function embezzlerCount(): number {
 }
 
 export function estimatedTurns(): number {
+  // Assume roughly 2 fullness from pantsgiving and 8 adventures/fullness.
+  const pantsgivingAdventures = have($item`Pantsgiving`)
+    ? Math.max(0, 2 - get("_pantsgivingFullness")) * 8
+    : 0;
+  const nightcapAdventures = globalOptions.ascending && myInebriety() <= inebrietyLimit() ? 60 : 0;
+  const thumbRingMultiplier = have($item`mafia thumb ring`) ? 1.04 : 1;
```
Comment (horrible-little-slime) [inline src/embezzler.ts:324]: I feel the need to say it's actually 1/0.96 but I think we can call that negligible

Comment (phulin) [reply src/embezzler.ts:324]: yep

File: src/embezzler.ts (hunk 1 above)
Comment (phulin) [reply src/embezzler.ts:329]: OK

**Review by horrible-little-slime — APPROVED** [review]

**Pull request merged**